### PR TITLE
Change NaN values to -1

### PR DESCRIPTION
**Pull request by jfewii**

### CODE REVIEW

File: postal-injection/src/test/java/com/google/sps/servlets/OptimalShippingMethodClusterMapTest.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.ClusterInfo;
+import com.google.sps.data.ZipCodeSummaryRow;
```
Comment (k-sharpe) [inline postal-injection/src/test/java/com/google/sps/servlets/OptimalShippingMethodClusterMapTest.java:4]: I dont think this import works anymore, data was changed to dto and other classes got moved. Try pulling master and it should show you what to change

**Comment by jennygaoo** [comment]
why don't the files compile?

File: postal-injection/src/test/java/com/google/sps/servlets/OptimalShippingMethodClusterMapTest.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    ZIPCODE_WAREHOUSE_LIST_2.add(ZIPCODE_SUMMARY_ROW_SINGLE_WAREHOUSE_5);
+    HashMap<String, ArrayList<ZipCodeSummaryRow>> ZIP_CODE_SUMMARY_MAP =
+        new HashMap<String, ArrayList<ZipCodeSummaryRow>>();
+    ZIP_CODE_SUMMARY_MAP.put(ZIPCODE_1, ZIPCODE_WAREHOUSE_LIST_1);
+    ZIP_CODE_SUMMARY_MAP.put(ZIPCODE_2, ZIPCODE_WAREHOUSE_LIST_2);
+    ArrayList<ClusterInfo> EXPECTED_ALL_CLUSTER_INFO = new ArrayList<ClusterInfo>();
+    EXPECTED_ALL_CLUSTER_INFO.add(EXPECTED_CLUSTER_INFO_OBJECT_POSTAL);
+    EXPECTED_ALL_CLUSTER_INFO.add(EXPECTED_CLUSTER_INFO_OBJECT_DIRECT);
+    EXPECTED_ALL_CLUSTER_INFO.add(EXPECTED_CLUSTER_INFO_OBJECT_DIRECT_2);
+    EXPECTED_ALL_CLUSTER_INFO.add(EXPECTED_CLUSTER_INFO_OBJECT_POSTAL_2);
```
Comment (andreahu12) [inline postal-injection/src/test/java/com/google/sps/servlets/OptimalShippingMethodClusterMapTest.java:124]: local variables should be lowercase like zipcodeWarehouseList1, zipcodeWarehouseList2, zipcodeSummaryMap, expectedAllClusterInfo, etc.

File: postal-injection/src/test/java/com/google/sps/servlets/OptimalShippingMethodClusterMapTest.java [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    ArrayList<ClusterInfo> allClusterInfo =
+        optimalShippingMethodClusterMap.getAllClusterInfo(ZIP_CODE_SUMMARY_MAP);
+    Assert.assertEquals(allClusterInfo, EXPECTED_ALL_CLUSTER_INFO);
```
Comment (andreahu12) [inline postal-injection/src/test/java/com/google/sps/servlets/OptimalShippingMethodClusterMapTest.java:127]: nit: add newlines between test setup, action being tested, and assertion to make it visually clear

**Pull request merged**